scene/register_scene_types.cpp

```diff
@@ -616,6 +622,7 @@ void register_scene_types() {
 	GDREGISTER_CLASS(RootMotionView);
 	GDREGISTER_VIRTUAL_CLASS(SkeletonModifier3D);
 	GDREGISTER_CLASS(RetargetModifier3D);
+#ifndef PHYSICS_3D_DISABLED
```

Comment:
We're losing a `yield` in this conditional block. Not sure whether they're really needed but there should probably still be one in all cases. Maybe move it before this block?